### ISSUE TITLE
ui idea: add some "delimiter" which will create "new line"

### ISSUE THREAD

**Issue by yosefy** [bugy/script-server#182]
it is bit strange that they all the same exactly almost
may be right name is grouping?

so the rest of params will be forced to move to next line

on UI:
 ---------     -----------
| paraam 1 |  | param2 |
 ---------     -----------
==================
 ---------
| paraam 3 |
 ---------

in config.json
just to explain the idea:

{
  name: param1

},
{
  name: param2,
  type: text
},
{
  name: None
  value: "="
  type: delimiter (i guess)
},
{
  name: param3
}

**Comment by bugy** [comment]
HI @yosefy, thanks for the proposal. 
Do you want to use this delimeter for some kind of grouping? 
I prefer not to occupy vertical space, if possible

Would put this ticket on voting. If some people would need it as well, would be happy to implement.

**Comment by CaseyLabs** [comment]
Plus one to this idea. I'm testing a very simple script for launching AWS EC2 instances, and would prefer 1 parameter per line (as I'm only passing in 3 parameters at the moment).

**Comment by tristanlatr** [comment]
I agree that would be useful, some scripts can take many many params.
The delimiter could have a title (and be rendered as a section containing params utils next delimiter is found) to add even more sens and structure.

**Comment by bugy** [comment]
This is also configurable via admin UI, if the syntax is confusing
